tools/Az.Tools.Predictor/Az.Tools.Predictor/Az.Tools.Predictor.ps1

```diff
@@ -0,0 +1,23 @@
+$version = $PSVersionTable.PSVersion
+
+function PSConsoleHostPredictor
+{
+    Microsoft.PowerShell.Core\Set-StrictMode -Off
```

Comment:
You are assuming the `PSReadLine` will always be the version that you are expecting, which may not be true. In that case, calling like `Set-PSReadLineOption -PredictionSource Plugin` and `Set-PSReadLineOption -PredictionViewStyle InlineView` will fail. It's okay to do so for testing purpose, but not for the official module.

Reply:
Yes. This script is to make it's possible to test the experience out of box. We need to address in long term what is the best way to enable the correct options if they ever change. Do you have any suggestion about enabling those options?

Reply:
For the official module, it shouldn't interfere with the PSReadLine setting. The user controls the PSReadLine setting -- deciding what source to use and what view to use. Documentation is needed to show how it works end-to-end.
The module could write out a verbose message when loading to point to the documentation.

Reply:
Thanks. Let me remove this file. I think we can prepare the script separately for the testing.